action.yaml

```diff
@@ -19,6 +19,9 @@ inputs:
   buildNumber:
     description: If set, will be used for the build number. (Typically this is read from process.env.GITHUB_RUN_NUMBER.)
     required: false
+  buildNumberOffset:
```

Comment:
Should we document this input in the [README](https://github.com/guardian/actions-riff-raff/blob/main/README.md#available-inputs)?

Reply:
Yes absolutely, my oversight.